web-marketplace/src/pages/Projects/Projects.SideFilter.tsx

```diff
@@ -127,15 +179,16 @@ export const ProjectsSideFilter = ({
                                 ...creditClassSelectedFilters,
                                 [path]: event.target.checked,
                               });
-                              track<
-                                'filterCreditClass',
-                                FilterCreditClassEvent
-                              >('filterCreditClass', {
-                                creditClassId: path,
-                                selected: getFilterSelected(
-                                  event.target.checked,
-                                ),
-                              });
+                              if (path !== UNREGISTERED_PATH)
```

Comment:
maybe we want to start tracking users clicking on "unregistered projects" filter, although I don't think we had that for the old "show off chain projects" filter cc/ @erikalogie